src/main/java/ast/ASTNodes.kt

```diff
@@ -2,6 +2,12 @@ package ast
 
 interface ASTNode
 
-data class Child(var father: ASTNode, var child: ASTNode) : ASTNode
-data class Node(var name: String) : ASTNode
-data class Sibling(var siblings: List<ASTNode>) : ASTNode
+data class Child(var father: ASTNode, var child: ASTNode) : ASTNode {
+    override fun toString(): String = "$father\n${child.toString().prependIndent("    ")}"
```

Comment:
Can we add this `"    "` as a constant at the top? Same for all hard coded  strings in files bellow.

Reply:
Complete